### ISSUE TITLE
idea: things to do if a version isn't found

### ISSUE THREAD

**Issue by jhheider** [pkgxdev/brewkit#91]
- run `pkg inv ${pkg}`
- run `gum confirm 'Would you like to open an issue to build ${pkg}${versionspec}?' && gh create issue -R teaxyz/pantry -t "${pkg}${versionspec}" -b "Please build ${pkg}${versionspec}!" -l "version-request"`

**Comment by mxcl** [comment]
you mean when we don't find a dependency during the test or build steps?

**Comment by jhheider** [comment]
I think this was about runtime version location. Though I think last I looked, it does suggest available versions now.

**Comment by mxcl** [comment]
but this is the brewkit repo

**Comment by jhheider** [comment]
Yes.... Clearly I failed to capture my personal context. Must have been while building...

**Comment by mxcl** [comment]
well I think yes, it makes sense to error well if you are trying to install a pkg that isn't available locally or remotely via dependencies for build or test or the rest. So… approved! lol
